gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -254,7 +256,8 @@ public LeaseAttemptStatus tryAcquireLease(DagActionStore.DagActionLeaseObject da
         log.debug("tryAcquireLease for [{}, is; {}, eventTimestamp: {}] - CASE 1: no existing row for this dag action,"
             + " then go ahead and insert", dagActionLeaseObject.getDagAction(),
             dagActionLeaseObject.isReminder() ? "reminder" : "original", dagActionLeaseObject.getEventTimeMillis());
-        int numRowsUpdated = attemptLeaseIfNewRow(dagActionLeaseObject.getDagAction());
+        int numRowsUpdated = attemptLeaseIfNewRow(dagActionLeaseObject.getDagAction(),
+            ExponentialBackoff.builder().maxRetries(3).initialDelay(10L).build());
```

Comment:
Can we set the defaults as static variables and include them with units? If it's just 10ms delay it wouldn't be much more than a jitter

Reply:
I set defaults for the static variable but I do actually want it to function as jitter for retrying a deadlock exception so I added some randomness to the initial delay. There may be other exceptions we want larger delay for, what do you think as (30,60ms) for the value?

Reply:
increased to be in range 20,220

Reply:
Sounds good